src/validator.ts

```diff
@@ -414,6 +468,15 @@ const isBooleanOrUnknownType = (node: Expression): boolean => node.match({
   Expression: _ => false,
 })
 
+const valueFor: any | undefined = (node: Expression | Assignment) =>
+  node.match({
+    Literal: node => node.value,
+    Return: node => valueFor(node.value),
+    Expression: _ => undefined,
```

Comment:
No me molesta para nada si queda así, pero just FYI, también podés usar la category `Node` como una suerte de `otherwise`:
```
Node: _ => undefined
```

Reply:
👍🏼  done!